### PR TITLE
New org perms

### PR DESCRIPTION
**Pull request by CoasterFan5**

### CODE REVIEW

**Review by LeoDog896 — CHANGES_REQUESTED** [review]

File: src/routes/(app)/org/[id=integer]/settings/members/+page.svelte [hunk 1]
```diff
@@ -64,6 +57,8 @@
 			showingModal: 'banMember'
 		});
 	};
+
+	let searchBox: SearchBox<(typeof data)['roles'][number]>;
```
[the remark this answers is not in the document]
Comment (CoasterFan5) [reply src/routes/(app)/org/[id=integer]/settings/members/+page.svelte:61]: I don't know what you mean here.

File: src/routes/(app)/org/[id=integer]/club/[clubId]/+layout.server.ts [hunk 2]
```diff
@@ -31,6 +34,20 @@ export const load = async ({ params, parent }) => {
 		error(404, 'Club Not Found');
 	}
 
+	const orgUser = user
+		? await prisma.orgUser.findFirst({
+				where: {
+					AND: {
```
[the remark this answers is not in the document]
Comment (CoasterFan5) [reply src/routes/(app)/org/[id=integer]/club/[clubId]/+layout.server.ts:40]: Yes

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+	"cSpell.words": ["orgid"]
```
[the remark this answers is not in the document]
Comment (CoasterFan5) [reply .vscode/settings.json:2]: Funnily, you made a typo in the link: https://github.com/crate-ci/typos
*using this as a personal note

File: src/routes/(app)/org/[id=integer]/club/[clubId]/settings/members/+page.server.ts [hunk 4]
```diff
@@ -1,57 +1,67 @@
...
 		club,
 		user,
-		clubUser
+		canTransferOwner
 	};
 };
 
 export const load = async ({ cookies, params }) => {
-	await validateUser(cookies.get('session'), params);
+	const { canTransferOwner } = await validateUser(cookies.get('session'), params);
```
[the remark this answers is not in the document]
Comment (CoasterFan5) [reply src/routes/(app)/org/[id=integer]/club/[clubId]/settings/members/+page.server.ts:64]: What does this even mean?

[the remark this answers is not in the document]
Comment (LeoDog896) [reply src/routes/(app)/org/[id=integer]/club/[clubId]/settings/members/+page.server.ts:64]: good job

File: src/routes/(app)/org/[id=integer]/settings/members/+page.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (LeoDog896) [reply src/routes/(app)/org/[id=integer]/settings/members/+page.svelte:61]: good job

**Review by LeoDog896 — APPROVED** [review]

**Pull request merged**